instrumentation/vertx/vertx-sql-client-4.0/javaagent/build.gradle.kts

```diff
@@ -12,13 +12,31 @@ muzzle {
 }
 
 dependencies {
-  library("io.vertx:vertx-sql-client:4.0.0")
-  compileOnly("io.vertx:vertx-codegen:4.0.0")
+  val version = "4.0.0"
+  library("io.vertx:vertx-sql-client:$version")
+  compileOnly("io.vertx:vertx-codegen:$version")
 
   testInstrumentation(project(":instrumentation:netty:netty-4.1:javaagent"))
 
-  testLibrary("io.vertx:vertx-pg-client:4.0.0")
-  testLibrary("io.vertx:vertx-codegen:4.0.0")
+  testLibrary("io.vertx:vertx-pg-client:$version")
+  testLibrary("io.vertx:vertx-codegen:$version")
+}
+
+testing {
+  suites {
+    val testVertx5 by registering(JvmTestSuite::class) {
```

Comment:
Misleading suite name. Name indicates that version 5 is tested, really version 4.5 is tested. Why do you need tis suite at all? Don't latest dep tests already test `4.5.13`?

Reply:
> misleading

agree

> need at all

the base test uses `io.vertx:vertx-sql-client:4.0.0` - so I think we do need this